src/features/surveys/managers/surveyAnswerManager.ts

```diff
@@ -171,5 +172,6 @@ export const useSurveyAnswerManager = (initialData: SurveyWithQuestions) => {
     activeQuestionIndex,
     handleNextQuestion,
     handlePreviousQuestion,
+    QuestionLength,
```

Comment:
We don't need export additional value here, since we are exporting `formData`. We can use just `formData.questions.length`.

Reply:
I don't know why, but my first idea was to extract this value from formData but i was null all the time, thats why i went this way by outputing QuestionLength instead.

Reply:
That property has all data about questions, so since questions are displayed correctly I guess it should have proper data :)